defaults/main.yml

```diff
@@ -3,6 +3,7 @@
 storage_provider: "blivet"
 storage_use_partitions: null
 storage_disklabel_type: null  # leave unset to allow the role to select an appropriate label type
+storage_safe_mode: false  # do not remove devices as needed to make space for new volumes
```

Comment:
Does the role remove existing volumes to make space for new ones? I.e. if you have a pool (VG) which is completely used by a volume oldlv, and you try to create a volume newlv, does the role remove oldlv to create space for newlv?

Reply:
No.

Reply:
That's good to hear, but the comment above sounds like it does.

Reply:
...but I see your point. I'll change the wording. I wrote that comment before I wrote the actual code changes.